oas

```diff
@@ -1,506 +1,354 @@
-        "description": "This defines the structure of an error payload when one is present. Actual messages and additional error information content may change from one implementation to another",
-        "type": "object",
-        "required": [
-          "message"
-        ],
-        "properties": {
-          "message": {
-            "type": "string",
-            "description": "Error description.",
-            "example": "General description"
-          },
-          "errors": {
-            "description": "Additional detailed information",
-            "type": "array",
-            "items": {
-              "type": "string"
+            "AssetDataResponse": {
+                "title": "AssetDataResponse",
+                "type": "object",
+                "properties": {
+                    "identifier": {
+                        "type": "string",
+                        "description": "Physical identifier (MAC Address) of the returnable asset",
+                        "example": "12:34:56:78:90:AB"
+                    },
+                    "data": {
+                        "type": "array",
+                        "items": {
+                            "$ref": "#/components/schemas/SensorData"
+                        }
+                    }
+                },
+                "description": "Data model for the data response"
+            },
+            "GPS": {
+                "title": "GPS",
+                "type": "object",
+                "properties": {
+                    "lat": {
+                        "type": "number",
+                        "description": "Latitude, a coordinate that specifies the north-south position of a point",
+                        "example": 48.5
+                    },
+                    "long": {
+                        "type": "number",
+                        "description": "Longitude, a coordinate that specifies the east-west position of a point",
+                        "example": 25.3
+                    }
+                },
+                "description": "Data Model for GPS"
+            },
+            "Measurements": {
+                "title": "Measurements",
+                "type": "object",
+                "properties": {
+                    "temperature": {
+                        "type": "number",
+                        "description": "Measured temperature of the asset's temperature sensor",
+                        "example": 29.5
+                    },
+                    "humidity": {
+                        "type": "number",
+                        "description": "Measured humidity of the asset's humidity sensor",
+                        "example": 50.1
+                    },
+                    "pressure": {
+                        "type": "number",
+                        "description": "Measured pressure of the asset's pressure sensor",
+                        "example": 1020
+                    },
+                    "gps": {
```

Comment:
Should not be handled as a measurement and rather as a "top-level" attribute next to timestamp?